free-3.md

```diff
@@ -4,51 +4,71 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+I'd like to see if it is possible to extend the work of Milo Han from last semester. He (or she?) wrote a program for sight reading. The language would serve music teachers and their students in improving their sight reading. A language might be good as it would be able to specify what kind of music one wants to sight read in order to sharpen one's weaknesses.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Sight reading is the process of reading music without having seen the music before. Once someone reads through the music once, the music cannot be reused for sight reading. As a result, the need for a constant source of new music would be satisfied with a DSL for sight reading. We are helping music students and teachers. The current experience is to find music from the books and play through them. However, the books may contain music that is too easy or too hard. Therefore, music teachers and students tend to filter through resources to get good sight-reading material. By then, there might be little to none to help them. With our DSL, it would solve this problem and let students quickly gain skill in sight reading.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because we want the users to give input on what kind of sheet music is generated for sight reading. A language can express a lot, and this kind of detail can create sheet music tailored to individual students and their needs. It addresses the problem of finding the right kind of music to sight read.
```

Comment:
So basically you want to generate stylistically and thematically appropriate music for the sake of sight reading? Cool, but I'm curious to see how you specify that with a language.